mxcube3/ui/components/RemoteAccess/ObserverDialog.jsx

```diff
@@ -66,7 +72,7 @@ export class ObserverDialog extends React.Component {
           <FormControl
             inputRef={(ref) => { this.name = ref; }}
             type="text"
-            placeholder="Your name"
+            default={this.props.loginInfo.selectedProposal}
```

Comment:
do we have anywhere the loginID? So that it displays username or proposal based on logintype.

Reply:
You probably know that better than me, there is maybe something in loginInfo. We are still not using user based accounts so did not really look into that

Reply:
okey, not big deal. I can fix that later